pkg/osvscanner/vulnerability_result.go

```diff
@@ -69,6 +69,25 @@ func buildVulnerabilityResults(
 				}
 			}
 		}
+
+		for _, vuln := range pkg.Vulnerabilities {
+			if isUnimportant(pkg.Package.Ecosystem, vuln.Affected) {
+				for i, group := range pkg.Groups {
+					if slices.Contains(group.IDs, vuln.ID) {
+						if group.ExperimentalAnalysis == nil {
+							pkg.Groups[i].ExperimentalAnalysis = make(map[string]models.AnalysisInfo)
+						}
+						// Set unimportant vulns as uncalled
+						pkg.Groups[i].ExperimentalAnalysis[vuln.ID] = models.AnalysisInfo{
+							Called: false,
```

Comment:
nit: `Called` is misleading here, because it implies we did reachability analysis.

Is it possible for us to add a new field here to be clearer? I.e. just `Unimportant` ?

Reply:
Added an `Unimportant` configuration option under `ExperimentalAnalysis`. One concern is that this will explicitly assign `Unimportant` to false for any other records that have call analysis results in the JSON output (see changes in `internal/output/__snapshots__/machinejson_test.snap`). I'm reusing the `IsCalled()` function to check if a vulnerability is important. I try to avoid adding a new function to prevent direct changes to `pkg/`.